src/main/java/unicash/ui/StatusBarFooter.java

```diff
@@ -94,9 +93,11 @@ public void updateBalance(ObservableList<Transaction> transactions) {
         }
 
         if (Double.compare(balance, 0) < 0) {
-            balanceString = NEGATIVE_BALANCE_STRING + new Amount(Math.abs(balance));
+            double formattedNegativeBalance = (Math.abs(balance) * 100.0 / 100.0);
```

Comment:
Hm, wouldn't `* 100.0 / 100.0` just be `* 1` haha, can probably just use `Math.abs(balance)`